posthog/temporal/workflows/snowflake_batch_export.py

```diff
@@ -55,6 +73,33 @@ class SnowflakeInsertInputs:
     data_interval_end: str
 
 
+def put_file_to_snowflake_table(cursor, file_name: str, table_name: str):
```

Comment:
Is there anything to be gained from typing the `cursor` here?

Reply:
Not much, since it's kind of given it's the Snowflake cursor (that we don't even define ourselves). But, it's also easy to be diligent, so I'll add it